rpc/client/rpc_test.go

```diff
@@ -672,8 +673,8 @@ func TestBroadcastEvidenceDuplicateVote(t *testing.T) {
 		require.NoError(t, err)
 		client.WaitForHeight(c, status.SyncInfo.LatestBlockHeight+2, nil)
 
-		ed25519pub := ev.PubKey.(ed25519.PubKeyEd25519)
-		rawpub := ed25519pub[:]
+		ed25519pub := ev.PubKey.(ed25519.PubKey)
+		rawpub := tmbytes.HexBytes(ed25519pub)
```

Comment:
I'm a bit confused w.r.t. why this changed. Is it just a general RPC improvement?